### PR TITLE
Styles update

### PR DESCRIPTION
**Pull request by JakubJachacz**
Added some styling updates.

### CODE REVIEW

File: src/common/content/MainTile/index.js [hunk 1]
```diff
@@ -106,8 +106,8 @@ export const MainTile = ({
         <MovieRating>
           <StyledStarIcon />
           <Rate>
-            {`${rate.toFixed(1)} / `}
-            <MaxRate>10</MaxRate>
+            {`${rate.toFixed(1)} `}
+            <MaxRate> / 10</MaxRate>
```
Comment (AngelikaKrygier) [inline src/common/content/MainTile/index.js:110]: Great that you noticed that! :)

File: src/common/content/MainTile/styled.js [hunk 2]
```diff
@@ -203,11 +203,21 @@ export const MaxRate = styled.span`
   color: ${({ theme }) => theme.colors.darkerGrey};
 
   @media (max-width: ${({ theme }) => theme.breakPoints.phoneMax}) {
-    font-size: 13px;
+    display: none; 
   }
 `;
 
-export const Votes = styled(MaxRate)`
```
Comment (AngelikaKrygier) [inline src/common/content/MainTile/styled.js:210]: What do you think about it:

export const Votes = styled(MaxRate)`
  @media (max-width: ${({ theme }) => theme.breakPoints.phoneMax}) {
    font-size: 13px;
  }`

Using (MaxRate) you avoid code repetition :)

File: src/features/movies/MoviePage/MovieBanner/styled.js [hunk 3]
```diff
@@ -75,10 +75,12 @@ export const RateWrapper = styled.div`
 `;
 
 export const StyledStarIcon = styled(StarIcon)`
-  width: 40px;
+  width: 60px;
+  transform: scale(1.7);
```
Comment (AngelikaKrygier) [inline src/features/movies/MoviePage/MovieBanner/styled.js:79]: The "Star" file has some defect, re-uploading doesn't help. We need to hack it :)

**Review by AngelikaKrygier — APPROVED** [review]

File: src/features/movies/MoviePage/MovieBanner/styled.js (hunk 3 above)
Comment (RadekW86) [reply src/features/movies/MoviePage/MovieBanner/styled.js:79]: review the source svg file - check different svg exports from Figma

**Review by RadekW86 — APPROVED** [review]

File: src/common/content/MainTile/styled.js (hunk 2 above)
Comment (JakubJachacz) [reply src/common/content/MainTile/styled.js:210]: It used to be a styled(MaxRate), but additional stylings had no effect on it sadly, so I had to copy-paste required stylings and add my own and make it a styled.span instead.

**Pull request merged**